AEPMessaging/Tests/UnitTests/Messaging+EdgeEventsTests.swift

```diff
@@ -427,117 +427,209 @@ class MessagingEdgeEventsTests: XCTestCase {
+        // validate xdm map
         let dispatchedEventData = dispatchedEvent?.data
-        let dispatchedXdmMap = dispatchedEventData?[MessagingConstants.XDM.Key.XDM] as? [String: Any]
-        XCTAssertNil(dispatchedXdmMap?[MessagingConstants.XDM.IAM.IN_APP_MIXIN_NAME])
+        let dispatchedXdmMap = dispatchedEventData?["xdm"] as? [String: Any]
+        XCTAssertEqual("decisioning.propositionTrigger", dispatchedXdmMap?["eventType"] as? String)
+        let experienceMap = dispatchedXdmMap?["_experience"] as? [String: Any]
+        let decisioningMap = experienceMap?["decisioning"] as? [String: Any]
+        let propositionEventTypeMap = decisioningMap?["propositionEventType"] as? [String: Any]
+        XCTAssertEqual(1, propositionEventTypeMap?["trigger"] as? Int)
+        let propositionsArray = decisioningMap?["propositions"] as? [[String: Any]]
+        XCTAssertEqual(1, propositionsArray?.count)
+        let prop = propositionsArray?.first!
+        XCTAssertEqual("propId", prop?["id"] as? String)
+        XCTAssertEqual("propScope", prop?["scope"] as? String)
+        let scopeDetails = prop?["scopeDetails"] as? [String: Any]
+        XCTAssertEqual("mockCorrelationID", scopeDetails?["correlationID"] as? String)
+        let characteristics = scopeDetails?["characteristics"] as? [String: Any]
+        XCTAssertEqual("abcd", characteristics?["cjmEventToken"] as? String)
+        XCTAssertNil(characteristics?["cjmXdm"])
     }
-
-    func testSendExperienceEventInteractEventTypeNilInteraction() throws {
+    
+    func testSendPropositionInteractionNoScopeDetails() throws {
         // setup
         setConfigSharedState()
         setIdentitySharedState()
         let mockEvent = Event(name: "triggeringEvent", type: MessagingConstants.Event.EventType.messaging, source: EventSource.requestContent, data: nil)
         let mockEdgeEventType = MessagingEdgeEventType.inappInteract
-        let mockInteraction: String? = nil
+        let mockInteraction = "swords"
         let mockMessage = MockMessage(parent: messaging, event: mockEvent)
-
+                
         // test
-        messaging.sendExperienceEvent(withEventType: mockEdgeEventType, andInteraction: mockInteraction, forMessage: mockMessage)
-
+        messaging.sendPropositionInteraction(withEventType: mockEdgeEventType, andInteraction: mockInteraction, forMessage: mockMessage)
+        
         // verify
-        XCTAssertEqual(1, mockRuntime.dispatchedEvents.count)
-        let dispatchedEvent = mockRuntime.firstEvent
-        // validate no mixin in xdm map
-        let dispatchedEventData = dispatchedEvent?.data
-        let dispatchedXdmMap = dispatchedEventData?[MessagingConstants.XDM.Key.XDM] as? [String: Any]
-        XCTAssertNil(dispatchedXdmMap?[MessagingConstants.XDM.IAM.IN_APP_MIXIN_NAME])
+        XCTAssertEqual(0, mockRuntime.dispatchedEvents.count)
+    }
+    
+    func testSendPropositionInteractionInteractNoCharacteristics() throws {
```

Comment:
nit: is there a clearer way to name this test?

Reply:
probably - but this test will be going away with the same update mentioned above :)